code/__DEFINES/hud.dm

```diff
@@ -9,3 +9,12 @@
 #define HUD_STYLE_NOHUD 3 //! No hud (for screenshots)
 
 #define HUD_VERSIONS 3	//! Used in show_hud(); Please ensure this is the same as the maximum index.
+
+// Modes used by /obj/item/clothing/glasses/meson/engine
+
+
+#define MODE_NONE ""
+#define MODE_MESON "meson"
```

Comment:
This does not belong in the HUD defines.

Reply:
why... not? 

I was indirectly asked to make these strings global defines so that I wasn't checking against a string by Bacon in an earlier review, and they have to be defined before the check occurs as I learned. This is where all other global hud definitions are defined?

![image](https://user-images.githubusercontent.com/9547572/187683821-1654be83-7178-472e-81c7-d4b3b9e9b610.png)

Reply:
yes, you've put them in a bad spot, there are tons of defines files, there's got to be one that fits these better. This file seems to be for onscreen HUDs, not the worn kind.

Reply:
Would you prefer it be in a completely unique file? I can't keep them where they originally were even if I make them global because that file compiles *after* _wires.dm which results in this snippet of code failing to compile due to being undefined:

```
if(goggles.mode == MODE_TRAY)
	reveal_wires = TRUE
```

Reply:
Just looked for you,
sigh.dm would be a good spot. Borg mesons are there.
https://github.com/BeeStation/BeeStation-Hornet/blob/master/code/__DEFINES/sight.dm

Reply:
:+1:

Reply:
Done